presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -470,7 +470,16 @@ public PlanNode visitJoin(JoinNode node, RewriteContext<Void> context)
             PlanNode left = context.rewrite(node.getLeft());
             PlanNode right = context.rewrite(node.getRight());
 
-            return new JoinNode(node.getId(), node.getType(), left, right, canonicalizeJoinCriteria(node.getCriteria()), node.getFilter().map(this::canonicalize), canonicalize(node.getLeftHashSymbol()), canonicalize(node.getRightHashSymbol()));
+            return new JoinNode(
```

Comment:
it is better to separate formatting changes from actual changes. 
In that case it would be better to have a commit which splits the line. And commit which adds method arg.

I would just leave it one line for now. As this is common. And then maybe add separate commit which changes all `JoinNode()` calls to multi-line.